clients/src/test/java/org/apache/kafka/common/message/MessageTest.java

```diff
@@ -886,7 +888,6 @@ public void testProduceResponseVersions() throws Exception {
         testAllMessageRoundTrips(new ProduceResponseData()
             .setResponses(new ProduceResponseData.TopicProduceResponseCollection(singletonList(
                 new ProduceResponseData.TopicProduceResponse()
-                    .setName(topicName)
```

Comment:
Did we remove this for version compatibility? I would think name or ID would be returned in the response?

Reply:
Yes, topic name or id is added below based on the version.